lib/matplotlib/transforms.py

```diff
@@ -2436,8 +2463,8 @@ def __init__(self, boxin, boxout, **kwargs):
         Create a new :class:`BboxTransform` that linearly transforms
         points from *boxin* to *boxout*.
         """
-        assert boxin.is_bbox
-        assert boxout.is_bbox
+        if not boxin.is_bbox or not boxout.is_bbox:
+            msg = "'boxin' and 'boxout' must be bbox"
```

Comment:
Thanks for catching the `raise` missing. 

About the second comment: what do you mean with "true-ish". The `if` is actually testing that both instances have a `is_bbox` property, but I guess that then somewhere else the code is making use of some bbox method/property. Would you prefer something like this?

> "'boxin' and 'boxout' must look like a bbox"

Any suggestion?